### PR TITLE
Resolves #28 - Update SISP class to initialize url on constructor

### LINKED ISSUE #28: Update SISP class to initialize sispUrl on constructor
Since SISP provide different links to different versions, we'll be let the user provide his link to handle the payments

this ticket should: 
* Update Sisp constructor to initialize the sispUrl property 
* Update tests and make sure that everything is working

### PR DESCRIPTION
**Pull request by FabyoSK**
Since SISP provide different links to different versions, we'll be let the user provide his link to handle the payments

this ticket should: 
* Update Sisp constructor to initialize the url property : 
```js
  constructor({ posID, posAutCode, url }) {
    this.posID = posID;
    this.posAutCode = posAutCode;
    this.url = url;
  };
```
Example:
 ``` js
 new Sisp({
     posID: 900512,
     posAutCode: '123456789ssA',
     url: 'https://mc.vinti4net.cv/Client_VbV_v2'
 }); 
  ```
     
* Update tests and make sure that everything is working

### CODE REVIEW

**Review by FredSoares — DISMISSED** [review]

**Review by lcsfort — DISMISSED** [review]

File: src/index.test.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ describe('index', () => {
   it('should be exported correctly', () => {
     const sisp = new Sisp({
       posID: 900512,
-      posAutCode: '123456789ssA'
+      posAutCode: '123456789ssA',
+      sispUrl: 'https://mc.vinti4net.cv/Client_VbV_v2/biz_vbv_clientdata.jsp'
```
Comment (trismegis7us) [inline src/index.test.js]: Since we are already in the context of SISP let just call this `url`.

Comment (trismegis7us) [inline src/index.test.js]: Is this a real url? If so let's change it to an example instead.

**Review by lcsfort — DISMISSED** [review]

**Review by FredSoares — DISMISSED** [review]

**Review by trismegis7us — COMMENTED** [review]
The `README` needs to be updated

File: README.md [hunk 2]
```diff
@@ -22,8 +23,9 @@ const Sisp = require('sisp-payments');
 
 const posID = 900512;
 const posAutCode = "123456789ssA";
+const url = "https://mc.vinti4net.cv/payments'";
```
Comment (trismegis7us) [inline README.md]: You have an extra `'` at the end.

**Review by lcsfort — APPROVED** [review]

**Review by trismegis7us — APPROVED** [review]

**Review by savylopes — APPROVED** [review]

**Pull request merged**